src/ticks.jl

```diff
@@ -185,7 +185,8 @@ function optimize_ticks_typed(
     span_buffer,
     scale,
 ) where {F}
-    if (xspan = x_max - x_min) < eps(F)
+    xspan = x_max - x_min
+    if xspan < eps(F)
```

Comment:
Ah that's not important, I was tracking why different variables were type-unstable and wanted to highlight that `xspan` was created there because I overlooked it at first. I do think this style is more legible though, so we might just keep it?

Reply:
I always tend toward reducing lines and number of characters, if that is only code style, we should not modify it.